### ISSUE TITLE
[Doc] Add limitation to GlobeView

### ISSUE THREAD

**Issue by geraldo** [visgl/deck.gl#9190]
### Link

https://github.com/visgl/deck.gl/blob/master/docs/api-reference/core/globe-view.md#limitations

### Description

Actually GlobeView doesn't work with MapLibre (and I suspect Mapbox eather), which actually [states here](https://deck.gl/docs/developer-guide/base-maps/using-with-maplibre) that it works with MapView. But to make it clear we should add the folloding to [GlobeView Limitations](https://github.com/visgl/deck.gl/blob/master/docs/api-reference/core/globe-view.md#limitations):

> Only supports `MapView`

**Comment by Pessimistress** [comment]
You are correct that GlobeView does not work with Mapbox/Maplibre. But I'm not following your suggestion for the documentation. `GlobeView` and `MapView` are mutually exclusive - you can use one or another. Why would you mention MapView in GlobeView documentation?

**Comment by geraldo** [comment]
It wasn´t clear for me till now that using base maps like Mapbox/Maplibre is only supported in the `MapView`. I did understand the `View` concept like something you can use with any kind of layer.  But after reading again documentation on [Base Maps](https://deck.gl/docs/get-started/using-with-map) now this limitation seams clear. But maybe it could be stated clearly on that page, that Base Maps only work in `MapView`?

**Comment by Pessimistress** [comment]
Ok, I think we could add a section in the base map pages about alternative projection modes.

Just FYI we are working to support Maplibre's globe projection. Mapbox remains out of scope.

**Comment by geraldo** [comment]
> Just FYI we are working to support Maplibre's globe projection. Mapbox remains out of scope.

Should that work together with `GlobeView`? Testing the new [globe projection](https://maplibre.org/news/2024-09-30-maplibre-newsletter-september-2024/#maplibre-gl-js) of _maplibre-gl@5.0.0-pre.1_ release together with deck.gl in `GlobeView` I see that coordinates don't concur... Where can I find documentation of that development?

**Comment by Pessimistress** [comment]
> Should that work together with GlobeView?

Yes, that is the goal.

> Testing the new globe projection of maplibre-gl@5.0.0-pre.1 release together with deck.gl in GlobeView I see that coordinates don't concur

As I said, we are *working* on it.

**Comment by maxmclau** [comment]
Would anyone consider working on this (and https://github.com/visgl/deck.gl/issues/9199) on a bounty/contract basis? We'd love to make the switch from ArcGIS, but need working GlobeView with rotation, panning, etc. We are willing to pay a dev to get support for this working

**Comment by ibgreen** [comment]
https://actionengine.com
 Specializes in contract work on deck.gl, loaders.gl etc @belom88

**Comment by maxmclau** [comment]
> https://actionengine.com Specializes in contract work on deck.gl, loaders.gl etc [@belom88](https://github.com/belom88)

looks perfect, thanks. just sent them an email

**Comment by charlieforward9** [comment]
@maxmclau interested in the status here - there has been a bit of work to maplibre and globe integration recently. Have you explored the improvements? 

I also have strong need for enhanced GlobeView control (bearing, pitch - GoogleEarth-level smoothness) and wouldn't mind having a discussion with you on this before I get into it more. 

Also just looking to triage some older issues.
